src/main/java/com/dataloom/client/serialization/SerializationConstants.java

```diff
@@ -36,8 +36,10 @@ private SerializationConstants() {}
     public static final String ENTITY_DEFINITIONS_FIELD      = "entityDefinitions";
     public static final String ASSOCIATION_DEFINITIONS_FIELD = "associationDefinitions";
 
+    public static final String ACL_KEY                       = "aclKey";
```

Comment:
ACL_OBJECT_PATH is "aclKey", although we might want to rename that at some point since that's a bit confusing